packages/wonder-blocks-dropdown/components/multi-select.js

```diff
@@ -168,50 +175,75 @@ export default class MultiSelect extends React.Component<Props, State> {
         }
     }
 
-    getShortcuts(): Array<
-        React.Element<typeof ActionItem | typeof SeparatorItem>,
-    > {
+    getShortcuts(): Array<DropdownItem> {
         const {children, selectedValues, shortcuts} = this.props;
-        const numOptions = React.Children.count(children);
-        // TODO(sophie): translate for i18n
+
         if (shortcuts) {
-            return [
-                <ActionItem
-                    disabled={numOptions === selectedValues.length}
-                    key="select-all"
-                    label={`Select all (${numOptions})`}
-                    indent={true}
-                    onClick={() => this.handleSelectAll()}
-                />,
-                <ActionItem
-                    disabled={selectedValues.length === 0}
-                    key="select-none"
-                    label="Select none"
-                    indent={true}
-                    onClick={() => this.handleSelectNone()}
-                />,
-                <SeparatorItem key="shortcuts-separator" />,
-            ];
+            const numOptions = React.Children.count(children);
+
+            // TODO(sophie): translate for i18n
+            const selectAllDisabled = numOptions === selectedValues.length;
+            const selectAll = {
+                component: (
+                    <ActionItem
+                        disabled={selectAllDisabled}
+                        label={`Select all (${numOptions})`}
+                        indent={true}
+                        onClick={this.handleSelectAll}
+                    />
+                ),
+                focusable: !selectAllDisabled,
+                populatedProps: {},
+            };
+
+            const selectNoneDisabled = selectedValues.length === 0;
+            const selectNone = {
+                component: (
+                    <ActionItem
+                        disabled={selectNoneDisabled}
+                        label="Select none"
```

Comment:
Might want to add another i18n comment here too, just so we don't fix one but not the other!